_posts/en/meetings/2016-12-22-meeting.md

```diff
@@ -0,0 +1,125 @@
+---
+title: IRC meeting summary for 2016-12-22
+permalink: /en/meetings/2016/12/22/
+name: 2016-12-22-meeting
+type: meetings
+layout: page
+lang: en

```

Comment:
It is indeed. Very weird, couldn't fix it with a new commit from my linux machine either so I made a new PR for this.